src/unitxt/metrics.py

```diff
@@ -38,16 +38,25 @@ def process(self, instance: Dict[str, Any], stream_name: str = None) -> Dict[str
 
 # TODO: currently we have two classes with this name. metric.Metric and matrics.Metric...
 class Metric(ABC):
+    use_additional_inputs: bool = True
```

Comment:
what is the downside of always providing the additional inputs? in other words, while support setting this flag to False?

Reply:
Worth considering.  Let's discuss.